packages/forms/stories-core/json/fields/core-datalist.json

```diff
@@ -10,6 +10,11 @@
       "restrictedDatalist": {
         "type": "string",
         "enum": [ "Apple", "Pine[apple]", "Banana", "Cher[ry", "Lemo}n", "Grapefruit" ]
+      },
+      "keyValue": {
+        "type": "string",
+        "enum": [ "a", "p", "b", "c", "l", "g" ],
+        "enumNames": [ "Apple", "Pine[apple]", "Banana", "Cher[ry", "Lemo}n", "Grapefruit" ]
```

Comment:
Are we really supposed to support enumNames as exposed feature ?
IMO this is supposed to be in an old version use case, mapped with merge.js, so no example in the core storybooks

Reply:
But it's used in our backend. so yes, but until backend has moved to the new uiSchema.